pkgs/by-name/lu/lunacy/package.nix

```
@@ -0,0 +1,107 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, makeWrapper
+, autoPatchelfHook
+, zlib
+, libgcc
+, fontconfig
+, libX11
+, lttng-ust
+, icu
+, libICE
+, libSM
+, libXcursor
+, openssl
+, imagemagick }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "lunacy";
+  version = "9.3.3";
+
+  src = fetchurl {
+    url = "https://lcdn.icons8.com/setup/Lunacy_${finalAttrs.version}.deb";
+    hash = "sha256-boIbJA/UuiA3bmXbZXro3IhjFUh6JYsOEprq3n668xk=";
+  };
+
+  unpackCmd = ''
+    mkdir -p root
+    dpkg-deb -x $src root
+  '';
+
+  buildInputs = [
+    zlib
+    libgcc
+    stdenv.cc.cc
+    lttng-ust
+    fontconfig.lib
+
+    # Runtime deps
+    libICE
+    libSM
+    libX11
+    libXcursor
+  ];
+
+  nativeBuildInputs = [
+    dpkg
+    makeWrapper
+    autoPatchelfHook
+  ];
+
+  dontBuild = true;
+  dontStrip = true;
+
```

```suggestion
  appendRunpaths = map (pkg: (lib.getLib pkg) + "/lib") [ icu openssl stdenv.cc.libc stdenv.cc.cc ]; 
  
  runtimeDependencies = [                                                                                                                             
    # Runtime deps                                                                                                                                    
    libICE                                                                                                                                            
    libSM
    libX11
    libXcursor                                                                                                                                        
  ];           
```
this goes along with my comments about the installPhase and postFixup